src/components/SideMenu/SideMenu.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```

Comment:
Ik ben niet zo'n fan van de jsx extensie, aangezien het in feite gewoon javascript is.